app/controllers/students_controller.rb

```diff
@@ -0,0 +1,76 @@
+class StudentsController < ApplicationController
+  before_action :set_student, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @students = Student.all
+  end
+
+  def show
+  end
+
+  def new
+    @student = Student.new
+  end
+
+  def edit
+  end
+
+  def create
+    @student = Student.new(student_params)
+
+    respond_to do |format|
+      if @student.save
+        format.html do
+          redirect_to @student, notice: 'Student was successfully created.'
```

Comment:
So `rails g scaffold` doesn't follow [ruby styleguide conventions](https://github.com/bbatsov/ruby-style-guide). I had to do some realignment in order to pass RuboCop.